packages/build/package.json

```diff
@@ -59,7 +59,7 @@
   "dependencies": {
     "@mongodb-js/devtools-github-repo": "^1.0.1",
     "@mongodb-js/mongodb-downloader": "^0.2.7",
-    "@mongodb-js/dl-center": "^1.0.1",
+    "@mongodb-js/dl-center": "^0.2.0",
```

Comment:
It's an "older" version, because the devtools-shared had older versions. We should be cautious when upgrading in the future.

Reply:
We should at the very least:
- Archive https://github.com/mongodb-js/dl-center so that it’s clear that no development happens there anymore
- Bump the version of dl-center in the devtools-shared to one that was not previously taken, otherwise we’ll get conflicts when we do a major bump of `@mongodb-js/dl-center` (similar to what we did for `mongodb-runner` where [we just bumped the major while adding it to devtools-shared](https://github.com/mongodb-js/devtools-shared/commit/64db843d198eb90b3b5f7a20cab062102df33e32#diff-56675ad02b7644df221605198cc4dadc830b8247e14e6fab7d58fd4899a165bcR16))

Reply:
If that's a versioning issue we only recently introduced, would it make sense to revert the publish and fix version in the shared repo instead of just keeping this as caveats?

Reply:
I agree with you, let's fix it 😄 

https://github.com/mongodb-js/devtools-shared/pull/151

Reply:
(going to unresolve this just so we don't forget about archiving the old repo, will remind Maurizio in our 1:1 which starts in 2 minutes :))